settings.py

```diff
@@ -22,3 +22,18 @@
 to be at least greater than FLOW_STATS and ideally it slightly greater than
 whichever longest network convergence FlowMods operations that your network has.
 """
+
+ACTION_PACES = {
```

Comment:
We also need pacing configs for `of_multi_table` and `telemetry_int` since you're replacing their old rolled out pacers.

Reply:
Added these in, still need to remove the batching on the source side of things.